task-definition.migration.json

```diff
@@ -106,6 +106,10 @@
                     "name": "CLICKHOUSE_PASSWORD",
                     "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:CLICKHOUSE_PASSWORD::"
                 },
+                {
+                    "name": "CLICKHOUSE_MIGRATION_HOSTS",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:CLICKHOUSE_MIGRATION_HOSTS::"
```

Comment:
Where do these come from?

Reply:
In general this section sets environment variables set in the ECS container running posthog production.

The value in `valueFrom` is a AWS secrets manager ARN which aws will resolve when creating the task. We keep our secrets in there.

In this case this is a list of hosts we host clickhouse on since we need to do the schema changes on all of them.